src/test_driver/openiotsdk/unit-tests/main/main.cpp

```diff
@@ -36,25 +36,25 @@ static void test_thread(void * argument)
     int status;
     CHIP_ERROR err;
 
-    ChipLogAutomation("Open IoT SDK unit-tests start");
+    printf("Open IoT SDK unit-tests start\r\n");
```

Comment:
Why do we move away from `ChipLogAutomation` to use `printf` instead ?

Reply:
In the shell unit test, we modify a bit the CHIP logger. It seems that this caused the line-splitting issue. I tried to use the print directly and it looks like it fixed the issue.

Reply:
Can you provide a link to the changes made to the logger in the test ?